### ISSUE TITLE
Trying to use \" in string gives an error

### ISSUE THREAD

**Issue by Jeremi360** [rakugoteam/Rakugo-Dialogue-System#257]
```
E 0:00:01:0811   Parser.gd:247 @ parse_script(): Parser: Error on line: 2, can not parse it !
  <C++ Source>   ./core/variant/variant_utility.cpp:905 @ push_error()
  <Stack Trace>  Parser.gd:247 @ parse_script()
                 Rakugo.gd:260 @ parse_script()
                 Rakugo.gd:302 @ parse_and_execute_script()
                 dialogue_UI.gd:45 @ start_dialogue()
                 intro_scene.gd:16 @ _ready()
```

with the out put line being
```
"This is a \"story\" of one heroine's adventure into wild jungles, uncharted caves, and untamed mountains."
```

@theLudovyc can you look at this ⤴️ ?
So this is strange as it looks  as for some reason our system thinks this is jump.

**Comment by theludovyc** [comment]
Yeah, it is because we do not handle \\" for now

**Comment by Jeremi360** [comment]
It can be fixed by #261
